sympy/vector/operators.py

```diff
@@ -0,0 +1,187 @@
+from sympy.core import Expr
+from sympy.vector.coordsysrect import CoordSysCartesian
+from sympy.vector.vector import Vector
+from sympy.vector.deloperator import Del
+
+
+class Gradient(Expr):
+    """
+    Represents unevaluated Gradient.
+
+    Examples
+    ========
+
+    >>> from sympy.vector import CoordSysCartesian, Gradient
+    >>> R = CoordSysCartesian('R')
+    >>> s = R.x*R.y*R.z
+    >>> Gradient(s, R)
+    Gradient(R.x*R.y*R.z, R)
+
+    """
+
+    def __new__(cls, scalar):
+        obj = Expr.__new__(cls, scalar)
+        obj._scalar = scalar
+        return obj
+
+    def _get_coord_sys_from_expr(self, scalar):
```

Comment:
I meant this one as a global function, not a class method. Let's avoid repeating the code. Also, this could be used in the operator functions.